new-dti-website/src/app/apply/page.tsx

```diff
@@ -9,54 +9,60 @@ import ApplyFAQ from '../../../components/apply/ApplyFAQ';
 import Banner from '../../../components/apply/Banner';
 import { isAppOpen } from '../../utils/dateUtils';
 
-const ApplyHero = () => (
-  <div className="text-[#FEFEFE] min-h-[calc(100vh-136px)] flex items-center relative">
-    <Banner
-      message={`We're no longer accepting applications for ${config.semester}. Stay tuned for opportunities next semester!`}
-      variant={'accent'}
-    />
-    <div
-      className="flex lg:flex-row xs:flex-col gap-x-[60px] lg:ml-[90px] lg:mr-[169px]
+const ApplyHero = () => {
+  const isApplicationOpen = isAppOpen();
+
+  return (
+    <div className="text-[#FEFEFE] min-h-[calc(100vh-136px)] flex items-center relative">
+      {!isApplicationOpen && (
```

Comment:
Thank you for catching this! Sorry about the oversight! Also thanks for renaming the variable name to the more specified isApplicationOpen!